web/packages/shared/setupTests.tsx

```diff
@@ -17,6 +17,22 @@
  */
 
 const crypt = require('crypto');
+const path = require('path');
+
+const failOnConsole = require('jest-fail-on-console');
+
+let entFailOnConsoleIgnoreList = [];
+try {
+  entFailOnConsoleIgnoreList = require('../../../e/web/testsWithIgnoredConsole');
+} catch (err) {
+  // Ignore errors related to teleport.e not being present. This allows OSS users and OSS CI to run
+  // tests without teleport.e.
+  const moduleNotFound =
+    err instanceof Error && err['code'] === 'MODULE_NOT_FOUND';
+  if (!moduleNotFound) {
+    throw err;
+  }
```

Comment:
This doesn't seem to actually catch the error when `teleport.e` is not present, hence why the CI check fails. I'll take care of it tomorrow.

It should work though when you run this branch and the teleport.e branch on your machine though. https://github.com/gravitational/teleport.e/pull/4101